api/v1beta1/tinkerbellmachine_types.go

```diff
@@ -89,12 +89,13 @@ type BootOptions struct {
 	// for getting the associated hardware into a CDROM booting state.
 	// A HardwareRef that contains a spec.BmcRef must be provided.
 	//
-	// The format of the ISOURL must be http://$IP:$Port/iso/:macAddress/hook.iso
+	// The format of the ISOURL must be http://$IP:$Port/iso/hook.iso
 	// The name of the ISO file must have the .iso extension, but the name can be anything.
 	// The $IP and $Port should generally point to the IP and Port of the Smee server
 	// as this is where the ISO patching endpoint lives.
-	// The ":macAddress" is a placeholder for the MAC address of the hardware and
-	// should be provided exactly as is: ":macAddress".
+	// The controller will append the MAC address of the hardware in the ISO URL
```

Comment:
This is a good start. Will you please add why "The controller will append the MAC address" is needed.